rsa/_compat.py

```diff
@@ -26,6 +26,8 @@
 MAX_INT32 = (1 << 31) - 1
 MAX_INT16 = (1 << 15) - 1
 
+PY3 = sys.version_info[0] >= 3
```

Comment:
Let's make this ` == 3`, as `PY3` being `True` on Python 4.x would be rather silly.